README.md

```diff
@@ -239,10 +95,6 @@ Make sure the secret is at least 30 characters and all random, no regular words
 or you’ll be exposed to dictionary attacks. You can use `rake secret` to
 generate a secure secret key.
 
-#### `SESSION_SECRET_KEY`
-
-This is used to sign the session used by the Sidekiq admin interface.
-
 #### `SMTP_USERNAME`, `SMTP_PASSWORD`
```

Comment:
I thought the same, but in fact the sendgrid API calls are still in public. They can be removed when we move email validation to a pvb2 API call (as discussed), or better yet remove entirely (as also discussed!).